.gitignore

```diff
@@ -6,6 +6,7 @@ _themes/
 _themes.MSDN.Modern/
 _themes.VS.Modern/
 .vscode
+.idea
```

Comment:
I would suggest not adding this as this repo is never meant to be opened by idea.

Reply:
I use Idea to open every Git repo.
Well, I'll remove this change.